README.md

```diff
@@ -222,6 +222,27 @@ This invokes QEMU on the image in `builds/latest`.  It uses `-snapshot`,
 so any changes are thrown away after you exit qemu.  To exit, type
 `Ctrl-a x`.  For more options, type `Ctrl-a ?`.
 
+### Running with customizations
+
+Using `coreos-assembler`, it is possible to run a CoreOS style qcow2 image
+that was built elsewhere.  To do this, you need to use the `kola qemuexec`
+subcommand and point to the qcow2 image in the current directory.  (If you
+do not use the `--devshell` option, you will likely be unable to login to
+the image without providing an Ignition config.)
+
+```
+$ cosa kola qemuexec --devshell --qemu-image ./coreos-style.qcow2
```

Comment:
This is really `cosa run --qemu-image ./coreos-style.qcow2`.
I see `--devshell` as kind of an implementation detail of `cosa run`.

Reply:
Makes sense; not sure how I didn't try this particular combination of commands/options.